### ISSUE TITLE
Is there a way to provide config file while running Hadolint as a docker image?

### ISSUE THREAD

**Issue by rakshithbr** [hadolint/hadolint#308]
- [ ] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Issue
While Hadolint is locally installed, config file (or --ignore, --format, etc.) can be used this way.
hadolint --config < config-file-name > < Dockerfile >

Is there a way to provide this flags when hadolint run as a docker image this way? 
docker run --rm -i hadolint/hadolint < Dockerfile>

In case, If I don't want to install hadolint, but also need to provide config values or output format, what can I do?

**Comment by devrck** [comment]
I have the same question how to do it?

**Comment by zemanlx** [comment]
@rakshithbr you can mount you config during docker run

Let's assume that your config and Dockerfile are in your current folder

```bash
$ ls
Dockerfile  hadolint.yaml  hooks  README.md
```
you can mount your current folder during docker run and then refer to your files from a container perspective

```bash
# descriptive version
docker run \
  --rm \
  --interactive \
  --volume "$PWD:/data:ro" \
  --workdir /data \
  hadolint/hadolint \
  /bin/hadolint \
    --config hadolint.yaml \
    Dockerfile

# or short version
docker run --rm -i -v "$PWD:/data:ro" -w /data hadolint/hadolint /bin/hadolint -c hadolint.yaml Dockerfile
```

**Comment by devrck** [comment]
For me this also worked:

```bash

$ docker run --rm -i \
		-v ${PWD}/.hadolint.yaml:/bin/hadolint.yaml \
		-e XDG_CONFIG_HOME=/bin hadolint/hadolint < Dockerfile
```

**Comment by rakshithbr** [comment]
Thanks @zemanlx. I did something similar. It worked for me.
